packages/core/src/components/pagination/pagination.scss

```diff
@@ -0,0 +1,50 @@
+.atom-pagination {
+  &__nav {
+    align-items: center;
+    display: flex;
+    font: var(--text-body-small);
+    gap: var(--spacing-xsmall);
+    letter-spacing: var(--text-body-small-letter);
+    list-style: none;
+
+    &-item {
+      background: none;
+      border: none;
+      color: var(--color-brand-secondary-regular);
+      cursor: pointer;
+      display: block;
+      font: var(--text-body-small);
+      height: var(--spacing-xxlarge);
+      letter-spacing: var(--text-body-small-letter);
+      padding: 0;
+      position: relative;
+      text-align: center;
+      width: var(--spacing-xxlarge);
+
+      &:not(&--ellipsis):hover {
+        background-color: var(--color-neutral-light-4);
+      }
+
+      &--active {
+        color: var(--color-neutral-regular);
+        font-weight: var(--font-weight-bold);
+
+        &::after {
+          background: var(--color-neutral-light-1);
+          border-radius: 9px;
+          bottom: 0;
+          content: '';
+          height: var(--spacing-xxsmall);
+          left: 0;
+          position: absolute;
+          width: 100%;
+        }
+      }
+
+      &--ellipsis {
+        cursor: auto;
+        height: 100%;
+      }
+    }
+  }
+}
```

Comment:
suggestion: it is not mandatory but as atomium uses scoped components we could use a direct approach instead `BEM`, for it we could use declarative approach for conditions, like `has-ellipsis` or `is-active`

Reply:
makes sense! done in [f5c26e5](https://github.com/juntossomosmais/atomium/pull/647/commits/f5c26e5cc7a55f273c0ef6a41213397439a72a12).